contrib/basicauth/nginx/registry.conf

```diff
@@ -0,0 +1,17 @@
+# Docker registry proxy for api version 2
+
+upstream docker-registry-v2 {
+  server registryv2:5000;
+}
+
+
+server {
+  listen 5441;
```

Comment:
Note that using 5441 may be easier since binding to port 80 comes with its own problems. Perhaps, we call out that point but it may be distracting.